artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/federation/AMQPFederationAddressConsumer.java

```diff
@@ -113,17 +113,20 @@ public class AMQPFederationAddressConsumer implements FederationConsumerInternal
    private AMQPFederatedAddressDeliveryReceiver receiver;
    private Receiver protonReceiver;
    private boolean started;
+   private final boolean largeMessageSync;
```

Comment:
Not needed to track this, this can be queried and not pollute the entire code base with copies of the configuration value.